### PR TITLE
Update embargo doc link in SECURITY_CONTACTS and change PST to PSC

### PR DESCRIPTION
**Pull request by joelsmith**
See https://github.com/kubernetes/security/issues/8 for more information

### CODE REVIEW

**Comment by joelsmith** [comment]
@cheftako I believe it's because it decouples the kubernetes from github. I think the idea is that if for some reason the org decided on using a different service for official git repos, the git.k8s.io docs links wouldn't have to be updated since that site could just redirect to the new repo locations. In my original link switch PR https://github.com/kubernetes/community/pull/3311, I was told to do it that way, so I did. Perhaps @nikhita can clarify.

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Defined below are the security contacts for this repo.
 #
-# They are the contact point for the Product Security Team to reach out
```
Comment (cheftako) [inline SECURITY_CONTACTS]: FYI this is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery.

**Comment by joelsmith** [comment]
@cheftako thanks, I'm also changing it there in https://github.com/kubernetes/kubernetes/pull/75201 so is it best to change it only there and wait for the sync or to change it in both places? Feel free to close the PR if we should wait for the sync.

File: SECURITY_CONTACTS (hunk 1 above)
Comment (ncdc) [reply SECURITY_CONTACTS]: I couldn't remember if this was a one-off or from staging. Thanks for clarifying. We can close this?

**Comment by cheftako** [comment]
@joelsmith Change it there and wait for sync is currently the correct procedure.

File: SECURITY_CONTACTS (hunk 1 above)
Comment (cheftako) [reply SECURITY_CONTACTS]: I believe so.

**Comment by joelsmith** [comment]
/close
The change was already synced by the Kubernetes Publisher bot from https://github.com/kubernetes/kubernetes/pull/75201

**Pull request closed without merging**